classes/Utils.php

```diff
@@ -759,23 +761,59 @@ public function get_course_completed_percent( $course_id = 0, $user_id = 0, $get
 		$totalContents    = $this->count( $course_contents );
 		$totalContents    = $totalContents ? $totalContents : 0;
 		$completedCount   = $completed_lesson;
+
+		$quiz_ids 		= array();
+		$assignment_ids = array();
+
+		foreach ( $course_contents as $content ) {
+			if ( 'tutor_quiz' === $content->post_type ) {
+				$quiz_ids[] = (int) $content->ID;
+			}
+			if ( 'tutor_assignments' === $content->post_type ) {
+				$assignment_ids[] = (int) $content->ID;
+			}
+		}
+
+		$quiz_ids_str		= QueryHelper::prepare_in_clause( $quiz_ids );
+		$assignment_ids_str = QueryHelper::prepare_in_clause( $assignment_ids );
+
+		global $wpdb;		
+		$quiz_completed = (int) $wpdb->get_var( 
+			$wpdb->prepare(
+				"SELECT count(quiz_id) completed 
+				FROM (
+					SELECT  DISTINCT quiz_id, course_id, attempt_status 
+					FROM 	{$wpdb -> tutor_quiz_attempts} 
+					WHERE 	quiz_id IN ({$quiz_ids_str}) 
+							AND user_id = % d 
+							AND attempt_status != 'attempt_started'
```

Comment:
**AND attempt_status != 'attempt_started'**
According to the clause above, an attempt whose status is **review_required OR attempt_timeout** will be counted as complete. I think it should have **AND attempt_status = 'attempt_ended'**